ConnectedCanteens/ngsild-payloads/BatchCreateEntities.json

```diff
@@ -53,9 +53,9 @@
                 "@value": "2020-06-30T14:00:00Z"
             }
         },
-        "isAlive": {
+        "dateLastValueReported": {
             "type": "Property",
-            "value": true,
+            "value": "2020-06-30T14:00:00Z",
```

Comment:
shouldn't it conform to Annex C.6 of the specification?

Reply:
You mean only for devices ? since we already agreed about this attribute for the EdgeGateway

Reply:
I mean the format of the date should conform to what is described in Annex C.6 (or so I guess)